src/Literature/Literatures.tsx

```diff
@@ -672,4 +672,51 @@ export const Literatures: Record<LiteratureName, Literature> = {
+        human body are able to be improved by technology: strength, speed, perception, intelligence. A large variety of
+        powerful augmentations have been developed, but for most they may as well not exist. How did things end up like
+        this?
+        <br />
+        <br />
+        Part of the answer is, unfortunately, that the exact technology that has allowed us to come so far in human
+        augmentation is also responsible for it's great expense and exclusivity.
+        <br />
+        The early days of human augmentation were plagued with issues. Immunosupressants were needed to keep people's
+        bodies from rejecting the foreign bodies we were implanting, with the body constantly trying to "restore" itself
+        to it's natural state. On top of that, with the messy nature of biology any replacement for a natural organ
+        would fall short of full functionality. Your arm bones don't just provide structural support and anchors for
+        muscles, they also help produce the blood in your body in the marrow. Even the fat in the body is a living organ
+        that produces hormones helping to regulate it's activity.
+        <br />
+        <br />
+        Thus, we were limited by the capacity of our bodies to accept and deal with these changes. Biology is redundant
+        and flexible, so replacing a few bone's won't impact the bodies functioning too badly. But every addition and
+        change would add up, ultimately severly limiting the body's capacity for augmentation.
+        <br />
+        <br />
+        Ultimately it was {CompanyName.VitaLife} who found a way around this limitation, though only at great expense...
+        They discovered that by using a radical new form of gene therapy to reshape a person's body to not only
+        accomodate but integrate the augmentations into their body at a cellular level, they could completely eliminate
+        the side-effects of augmenting.
+        <br />
+        <br />
+        The downside, of course, is the extreme expense and complexity of the process. Not only must each augment be
+        tailored for it's intended recipiant and a custom genetic therapy program developed, but to truely avoid any
+        side-effects this program must take into account the interactions between augmentations when multiple are
+        installed. The combinatoric explosion in complexity leads to exponentially increasing costs when more than one
+        augment is installed at the same time. Despite this most still prefer to install as many as possible at once,
+        because in addition the radical transforation their body undergoes leaves them relearning to use their body for
+        weeks or even months afterward.
+        <br />
+        <br />
+        In the end we're left with the theoretical capacity to transform ourselves to the limits of our imaginination,
```

Comment:
```suggestion
        In the end humanity is left with the theoretical capacity to transform themselves to the limits of their imaginations,
```
As above on first person